src/diff.js

```diff
@@ -41,8 +41,8 @@ export function render(diff) {
   }
 }
 
-export default function diffLogger(prevState, newState, logger, isCollapsed) {
-  const diff = differ(prevState, newState);
+export default function diffLogger(prevState, newState, logger, isCollapsed, customDiffer) {
+  const diff = (customDiffer || differ)(prevState, newState);
```

Comment:
@thiamsantos Yeah that makes sense -> https://github.com/evgenyrodionov/redux-logger/pull/241/commits/b1f3281a3392ea9381baf39b6cb2a83abfd161b3